dev-libs/cgreen/metadata.xml

```diff
@@ -2,6 +2,9 @@
 <!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
 	<!-- maintainer-needed -->
+	<use>
+		<flag name="libxml2">Use <pkg>dev-libs/libxml2</pkg> for output formatting in reports</flag>
```

Comment:
This is normally USE=xml.

Reply:
What is the simpler alternative here? You didn't explain that in the USE flag description nor in the commit message.

Reply:
Please explain this in the commit message and USE flag description.